src/requests/features.jl

```diff
@@ -317,3 +317,38 @@ function get_module_of(s::StaticLint.Scope, ms = [])
         return isempty(ms) ? "Main" : join(ms, ".")
     end
 end
+
+using Base.Docs, Markdown
+using Markdown: MD, HorizontalRule
```

Comment:
I think we have a convention in the codebase to have all `using` statements in the root `LanguageServer.jl` file.